packages/fxa-settings/src/pages/ResetPassword/ResetPasswordWithRecoveryKeyVerified/index.tsx

```diff
@@ -9,48 +9,63 @@ import { logViewEvent } from '../../../lib/metrics';
 import Ready from '../../../components/Ready';
 import { MozServices } from '../../../lib/types';
 import { REACT_ENTRYPOINT } from '../../../constants';
+import AppLayout from '../../../components/AppLayout';
+import { useFtlMsgResolver } from '../../../models';
 
 type ResetPasswordWithRecoveryKeyVerifiedProps = {
   serviceName?: MozServices;
+  // Verify if relier is Sync with the useRelier hook that will be implemented in FXA-6437
+  // instead of using a prop
```

Comment:
Should I hold on this PR until this hook is available?

Reply:
I would land it, considering FXA-6437 still isn't approved yet. There's a few follow ups as a result of FXA-6437. Can you add a follow up ticket for this?

Reply:
Sure thing. Created here: https://mozilla-hub.atlassian.net/browse/FXA-6804